javascript/gdx_getting_started_5.html

```diff
@@ -0,0 +1,96 @@
+<!--
+  This example uses the GDX JavaScript library to collect and manipulate data from a Vernier GoDirect Sensor. 
+  Press the 'Select GoDirect Device' button to open communication to the device in a Chrome browser.
+  Press the 'Download CSV' to save a copy of the data as a csv file.
+  
+  This program uses the following functions from the GDX Library:
+   
+  gdxDevice.selectDevice();
+  gdxDevice.close();
+  gdxDevice.sensors.filter();
+  gdxDevice.getSensor(int);
+  sensor.setEnabled(bool);
```

Comment:
Uses gdxDevice.start(). Does not use gdxDevice.sensors.filter()

Reply:
Done